### PR TITLE
documented phone home metrics

### PR DESCRIPTION
**Pull request by mooreds**
---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1205172787018319
  - https://app.asana.com/0/0/1205962910282590

### CODE REVIEW

File: astro/src/content/docs/get-started/download-and-install/system-requirements.mdx [hunk 1]
```diff
@@ -135,11 +135,27 @@ If you are running FusionAuth in an environment with no network access and are u
 
 If you are using the community plan, you can block all outbound network connectivity. Lack of connectivity will not affect the product. 
 
-Once on a paid plan, you must enable network access to FusionAuth, or your license will not function properly. Advanced features provided by FusionAuth Reactor cloud services require additional outbound network access.
+On a paid plan, you must enable network access to FusionAuth, or your features and license will not function properly.
```
Comment (mark-robustelli) [inline astro/src/content/docs/get-started/download-and-install/system-requirements.mdx:138]:
```suggestion
On a paid plan, you must enable network access to FusionAuth or the license and features will not function properly.
```

It is ok to leave as it, but I don't think the features are the users.  They are FusionAuth features.

File: astro/src/content/docs/get-started/download-and-install/system-requirements.mdx [hunk 2]
```diff
@@ -135,11 +135,27 @@ If you are running FusionAuth in an environment with no network access and are u
...
 
-If you are in a firewalled or restricted environment and have a paid plan, add the following hostnames to your network allow list:
+If you are in a firewalled or restricted environment and have a paid plan, add the following hostnames to your network allow list for proper functionality:
 
 * https://reactor.fusionauth.io
```
Comment (mark-robustelli) [inline astro/src/content/docs/get-started/download-and-install/system-requirements.mdx:142]: Is there a way to make these not clickable?  They are just a list of names to add and don't think they should be links.

File: astro/src/content/docs/get-started/download-and-install/system-requirements.mdx [hunk 3]
```diff
@@ -135,11 +135,27 @@ If you are running FusionAuth in an environment with no network access and are u
...
 * https://metrics.fusionauth.io
 * https://license.fusionauth.io
 
+### Collected Metrics
+
+Currently FusionAuth sends a payload to the metrics endpoint for all instances. Collected metrics include:
```
Comment (mark-robustelli) [inline astro/src/content/docs/get-started/download-and-install/system-requirements.mdx:148]: Do we have anything that says what we do with the data when collected?  I.e We don't send sell your data to mass marketers.  Only used for internal purposes.  That sort of thing.

**Review by mark-robustelli — APPROVED** [review]
No show stoppers just a few comments to consider.

File: astro/src/content/docs/get-started/download-and-install/system-requirements.mdx (hunk 3 above)
Comment (mooreds) [reply astro/src/content/docs/get-started/download-and-install/system-requirements.mdx:148]: It is all governed by the FusionAuth license, which is here: https://fusionauth.io/license

Comment (mooreds) [reply astro/src/content/docs/get-started/download-and-install/system-requirements.mdx:148]: I'll add a note.

**Review by mark-robustelli — APPROVED** [review]
LGTM

**Pull request merged**